src/app/components/client/CsatSurvey.tsx

```diff
@@ -0,0 +1,192 @@
+      dissatisfied: "https://survey.alchemer.com/s3/7714663/481c08f43d81",
+      neutral: "https://survey.alchemer.com/s3/7714663/6a94888b165c",
+      satisfied: "https://survey.alchemer.com/s3/7714663/996da64e2fbb",
+      "very-satisfied": "https://survey.alchemer.com/s3/7714663/668f6cb4d250",
+    },
+  },
+  {
+    id: "3-months",
+    timeThreshold: 90,
+    options: {
+      "very-dissatisfied":
+        "https://survey.alchemer.com/s3/7718223/9bf87045f7fb",
+      dissatisfied: "https://survey.alchemer.com/s3/7718223/4ebd39f49be3",
+      neutral: "https://survey.alchemer.com/s3/7718223/fe77a597f97a",
+      satisfied: "https://survey.alchemer.com/s3/7718223/fbbb597a762a",
+      "very-satisfied": "https://survey.alchemer.com/s3/7718223/8f7abc102a9a",
+    },
+  },
+  {
+    id: "6-months",
+    timeThreshold: 180,
+    options: {
+      "very-dissatisfied":
+        "https://survey.alchemer.com/s3/7718561/1354e1186d33",
+      dissatisfied: "https://survey.alchemer.com/s3/7718561/6dfb2e8b6d68",
+      neutral: "https://survey.alchemer.com/s3/7718561/2ff6ff90e603",
+      satisfied: "https://survey.alchemer.com/s3/7718561/9393c233103e",
+      "very-satisfied": "https://survey.alchemer.com/s3/7718561/a443cc84b78a",
+    },
+  },
+  {
+    id: "12-months",
+    timeThreshold: 351,
+    options: {
+      "very-dissatisfied":
+        "https://survey.alchemer.com/s3/7718562/c254fe9e3c33",
+      dissatisfied: "https://survey.alchemer.com/s3/7718562/8d2a7f93852f",
+      neutral: "https://survey.alchemer.com/s3/7718562/76e17004efd6",
+      satisfied: "https://survey.alchemer.com/s3/7718562/92b30b6aa491",
+      "very-satisfied": "https://survey.alchemer.com/s3/7718562/002e20b6b82f",
+    },
+  },
+] as const;
+
+type Props = {
+  elapsedTimeSinceInitialScan: number;
+  hasAutoFixedDataBrokers: boolean;
+  selectedTab: TabType;
+};
+
+const getRelevantSurvey = ({
+  elapsedTimeSinceInitialScan,
+  hasAutoFixedDataBrokers,
+  selectedTab,
+}: Props): Survey | undefined => {
+  const relevantSurvey = surveys.findLast(
+    (survey) =>
+      elapsedTimeSinceInitialScan >=
+      survey.timeThreshold * CONST_DAY_MILLISECONDS,
+  );
+
+  // Show the initial survey on the ”fixed” dashboard tab only to users
+  // that have autmatically fixed data broker results.
+  if (
+    relevantSurvey?.id === "initial" &&
+    (!hasAutoFixedDataBrokers || selectedTab !== "fixed")
+  ) {
+    return;
+  }
+
+  return relevantSurvey;
+};
+
+export const CsatSurvey = (props: Props) => {
+  const l10n = useL10n();
+  const [answer, setAnswer] = useState<keyof SurveyLinks>();
+
+  const recordTelemetry = useTelemetry();
+  const hasRenderedClientSide = useHasRenderedClientSide();
+  const survey = getRelevantSurvey(props);
+  const localDismissal = useLocalDismissal(`survey-csat_${survey?.id}`);
+
+  if (
+    !hasRenderedClientSide ||
+    typeof survey === "undefined" ||
+    (localDismissal.isDismissed && process.env.STORYBOOK !== "true")
```

Comment:
Good point: The reason why I added the exception for storybook here was so that a dismissed `CsatSurvey` would not prevent it from showing up to avoid confusion. Since this is the expected behaviour that might be fine, though. [Removed the condition for now](https://github.com/mozilla/blurts-server/pull/4421/commits/219f04d12c9e4f28e25158d925a87e4fea3dc7a9).